definitions/variable/utilisation.yaml

```
@@ -6,4 +6,8 @@
     unit: h/a
 - Utilisation|Heat Plant|Heat|{Heat Input}:
     description: utilisation / full load hours of heat plants producing heat from {Electricity Input}
+    unit: h/a
+- Utilisation|Storage|Discharging|Electricity|{Electricity Storage Types}:
```

I think that inverting the order would make more sense - it's a variable for utilisation -> of a storage technology -> for electricity -> for a particular kind of activity of that technology -> for a type of technology
```suggestion
- Utilisation|Storage|Electricity|Discharging|{Electricity Storage Types}:
```

---

Sounds good to me - I guess this hasn't been used anywhere yet...?

---

I implemented and committed the suggested changes